core/src/parking_lot.rs

```diff
@@ -175,14 +168,9 @@ where
 {
     // Try to read from thread-local storage, but return None if the TLS has
     // already been destroyed.
-    #[cfg(has_localkey_try_with)]
     fn try_get_tls(key: &'static LocalKey<ThreadData>) -> Option<*const ThreadData> {
         key.try_with(|x| x as *const ThreadData).ok()
     }
```

Comment:
There isn't much point keeping that as a separate function now.

Reply:
True. Removed.